warehouse/utils/utils.go

```diff
@@ -909,3 +909,18 @@ func GetLoadFileFormat(whType string) string {
 		return "csv.gz"
 	}
 }
+
```

Comment:
Not priority since we need to release it urgently. But it's a good contender for Unit tests. Can we add a TODO to create Unit tests for this.
wdyt?

Reply:
@Jayachand Shouldn't be too hard to write test cases. Unless you think otherwise, please add unit tests as well for this func